### PR TITLE
added custom render function. #197 fix

### PR DESCRIPTION
**Pull request by levgaas**
Ability to render emojis with custom function.
Function should return template to paste.

Please check if making `customEmoji` as function doesn't provide any collisions. I've done global search for `customEmoji` but didn't found a thing.

Maybe we should move regexp search for external function also. To be more versatile

### CODE REVIEW

File: src/js/modules/regex.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const regex = {
 	youtube      : /https?:\/\/(?:[0-9A-Z-]+\.)?(?:youtu\.be\/|youtube\.com(?:\/embed\/|\/v\/|\/watch\?v=|\/ytscreeningroom\?v=|\/feeds\/api\/videos\/|\/user\S*[^\w\-\s]|\S*[^\w\-\s]))([\w\-]{11})[?=&+%\w-]*/gi,
 	gmap         : /@\((.+)\)/gi,
 	twitter      : /https:\/\/twitter\.com\/\w+\/\w+\/\d+/gi,
-	smileys      : /(\:[\+\-\w]+\:|\<[\/\\]?3|[\(\)\\\D|\*\$][\-\^]?[\:\;\=]|[\:\;\=B8][\-\^]?[3DOPp\@\$\*\\\)\(\/\|])(?=\s|[\!\.\?]|$)/gi
+	smileys      : /(\:\w+\:)/gi
```
Comment (ritz078) [inline src/js/modules/regex.js]: `:+1:` is not acceptable in this regex

Comment (levgaas) [inline src/js/modules/regex.js]: Yeah. You are right. I'll try to figure smth out. Can you provide different examples of emojies?

Comment (levgaas) [inline src/js/modules/regex.js]: Try this `:[a-zA-Z0-9-_+]+:(:skin-tone-[2-6]:)?`

**Comment by ritz078** [comment]
Looks good.

**Comment by ritz078** [comment]
released as v4.1.13

**Pull request merged**